app.js

```diff
@@ -0,0 +1,84 @@
+const displayBook = document.querySelector('.display-books');
+const form = document.querySelector('form');
+
+class Books {
+  constructor() {
+    if (localStorage.getItem('books') === null) {
+      this.books = [];
+    } else {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  // add book methods
+  addBook(book) {
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  // remove book methods
+  removeBook(bookId) {
+    this.books = this.books.filter((item, index) => {
+      if (index !== bookId) {
+        return item;
+      }
+      return undefined;
+    });
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+}
+
+// create an object
+const listBooks = new Books();
+function onPageReload() {
+  displayBook.innerHTML = listBooks.books.map((book, index) => `
+  <div class="display">
```

Comment:
- [ ] Please change this function to the arrow function to follow ES6 syntax.

Reply:
done

Reply:
- [ ] As the previous reviewer mentioned, please change this function to make it an arrow function instead. Basically, what you need to do is get rid of the `function` keyword and do something like `const myFunction = () => { your code }`